Server/APIs/RoomsAPI.py

```diff
@@ -0,0 +1,110 @@
+from flask import Blueprint, request, jsonify
+from Model.Player import Player
+from Model.Room import Room
+from Model.RoomState import RoomState
+from Server.Database.DBMethods import DBMethods
+from Server.APIs.AuthenticationAPI import token_required
+import uuid
+from Common.RoomsAPIErrors import RoomsAPIErrors
+
+rooms_api = Blueprint('rooms_api', __name__)
+
+
+@rooms_api.route("/rooms", methods=['GET'])
+@token_required
+def get_rooms(current_player):
+    rooms_as_dicts = {}
+    index = 1
+    rooms_as_objects = get_available_rooms_as_objects()
+    for room_as_object in rooms_as_objects:
+        rooms_as_dicts[index] = room_as_object.to_dict()
+        index += 1
+    return jsonify(rooms_as_dicts)
```

Comment:
Why do you need to model it as dict? it forces you to use the indexes. 
{
  1: {name: "room A"},
  2: {name: "room B"}
}

json can contains array, you can send something like that: 
[{name: "room B"},{name: "room B"}]